.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/node:8
+      - image: circleci/node:14
```

Comment:
Could you check if docker would just run the deployment with node js v14 or also with lower node js versions? Maybe it's a little bit too high for the current use or?

Reply:
CI gave clear to node 14.

Reply:
which CI? My question was, if I have now currently v13.16.3 on my machine running, can I deploy this docker image?